jobs/templates/managed-service-broker-tests.yaml

```diff
@@ -0,0 +1,20 @@
+- job:
+    name: managed-broker-service-tests
+    project-type: pipeline
+    sandbox: true
+    pipeline-scm:
+      scm:
+        - git:
+            url: http://github.com/integr8ly/ci-cd
+            clean: true
+            branches: 
+              - master
+      script-path: scripts/msb-tests.groovy
+      lightweight-checkout: true
+    parameters:
+      - string:
+          name: CLUSTER_ROUTE
+          default: 'integr8ly-staging.dynamic.xpaas'
```

Comment:
Could you please add some description as well ?

Reply:
For both params if possible

Reply:
OK